go-controller/pkg/ovn/policy_unit_test.go

```diff
@@ -0,0 +1,106 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	goovn "github.com/ebay/go-ovn"
+	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
+	goovn_mock "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing/mocks/github.com/ebay/go-ovn"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAddAllowACLFromNode(t *testing.T) {
+	mockGoOvnNBClient := new(goovn_mock.Client)
+
+	tests := []struct {
+		desc                      string
+		inpSwName                 string
+		inpMgmtIp                 net.IP
+		errExp                    bool
+		errMatch                  error
+		onRetArgMockGoOvnNBClient []ovntest.TestifyMockHelper
+	}{
+		{
+			desc:      "test error when ovnNBClient.ACLAdd() fails",
+			inpSwName: "testSW",
+			inpMgmtIp: ovntest.MustParseIP("192.168.10.10"),
+			errMatch:  fmt.Errorf("ACLAdd() error when creating node acl for logical switch"),
+			onRetArgMockGoOvnNBClient: []ovntest.TestifyMockHelper{
+				{
+					OnCallMethodName: "ACLAdd", OnCallMethodArgType: []string{"string", "string", "string", "string", "int", "map[string]string", "bool", "string", "string"}, RetArgList: []interface{}{nil, goovn.ErrorExist},
+				},
+			},
+		},
+		{
+			desc:      "test when ACL already exists and confirm no error returned",
+			inpSwName: "testSW",
+			inpMgmtIp: ovntest.MustParseIP("192.168.10.10"),
+			onRetArgMockGoOvnNBClient: []ovntest.TestifyMockHelper{
+				{
+					OnCallMethodName: "ACLAdd", OnCallMethodArgType: []string{"string", "string", "string", "string", "int", "map[string]string", "bool", "string", "string"}, RetArgList: []interface{}{&goovn.OvnCommand{}, nil},
```

Comment:
I think you should expect the goovn.ErrorExist error here on the ACLAdd.

Reply:
@anfredette thanks, let me look into it